README.md

```diff
@@ -91,17 +92,33 @@ Available targets:
 | acm_primary_domain | A domain name for which the certificate should be issued | string | - | yes |
 | acm_san_domains | A list of domains that should be SANs in the issued certificate | list | `<list>` | no |
 | attributes | Additional attributes (e.g. `1`) | list | `<list>` | no |
-| chamber_format | Format to store parameters in SSM, for consumption with chamber | string | `/%s/%s` | no |
+| chamber_format | Format to store parameters in SSM, for consumption with `chamber` | string | `/%s/%s` | no |
 | chamber_service | `chamber` service name. See [chamber usage](https://github.com/segmentio/chamber#usage) for more details | string | `` | no |
 | delimiter | Delimiter to be used between `namespace`, `stage`, `name` and `attributes` | string | `-` | no |
+| documentdb_apply_immediately | Specifies whether any cluster modifications are applied immediately, or during the next maintenance window | string | `true` | no |
+| documentdb_cluster_enabled | Set to false to prevent the module from creating DocumentDB cluster | string | `true` | no |
+| documentdb_cluster_family | The family of the DocumentDB cluster parameter group. For more details, see https://docs.aws.amazon.com/documentdb/latest/developerguide/db-cluster-parameter-group-create.html | string | `docdb3.6` | no |
+| documentdb_cluster_parameters | List of DB parameters to apply | list | `<list>` | no |
+| documentdb_cluster_size | Number of DB instances to create in the cluster | string | `3` | no |
+| documentdb_enabled_cloudwatch_logs_exports | List of log types to export to CloudWatch. The following log types are supported: audit, error, general, slowquery | list | `<list>` | no |
+| documentdb_engine | The name of the database engine to be used for this DB cluster. Defaults to `docdb`. Valid values: `docdb` | string | `docdb` | no |
+| documentdb_engine_version | The version number of the database engine to use | string | `` | no |
+| documentdb_instance_class | The instance class to use. For more details, see https://docs.aws.amazon.com/documentdb/latest/developerguide/db-instance-classes.html#db-instance-class-specs | string | `db.r4.large` | no |
+| documentdb_master_password | (Required unless a `documentdb_snapshot_identifier` is provided) Password for the master DB user. Note that this may show up in logs, and it will be stored in the state file. Please refer to the DocumentDB Naming Constraints | string | `` | no |
```

Comment:
We don't autogenerate this?

Reply:
updated description (we do generate it if empty string provided)